revised/stellaswap-proposal

```diff
@@ -0,0 +1,237 @@
+* Efficiency rates of 10X - 20X
+* Low-slippage trading with much lower TVL requirements
+* Sustainable yields via trade fee generation
+* Launch of the first emissions-free farm (Astar)
+
+2) __Simplifying User Journey via Key Launches__ 
+We launched several features that aimed to simplify users’ DeFi journey to maximize our grants, These include;
+
+* [Fostering Cross-Chain swaps](https://twitter.com/StellaSwap/status/1620827821877886977) with Squid + Axelar, creating strong traction that led to greater capital inflows to Moonbeam. The top 2 out of 3 most popular paths for crosschain swaps was initiated towards Moonbea, from BNB and Polygon.
+* [XCM Transfers](https://twitter.com/StellaSwap/status/1587134387836944385) for native inter-parachain asset transfers
+* Alleviated native USDT acquisition limitation with [Meson](https://twitter.com/StellaSwap/status/1614660760701108225)
+* [Gas refunds program](https://twitter.com/StellaSwap/status/1592019268325249024) to foster greater trading
+* Developed universal router for auto-trade routing across all our AMMs
+
+3) __Igniting Moonbeam as the Hub__ 
+
+* Worked with Moonwell to establish [Moonbeam Ignite campaign](https://stellaswap.medium.com/introduction-to-moonbeam-ignite-liquidity-incentive-campaign-to-grow-ecosystem-liquidity-29e4d7629404) to spread the word on Moonbeam
+* Active participation in AMA’s with [PolkaWorld](https://twitter.com/polkaworld_org/status/1604134978933395457), [TeriyakiDon](https://twitter.com/MoonbeamNetwork/status/1598767216543076369), [DTMB](https://twitter.com/dtmb_xyz/status/1587881539567726593), [Polkawarriors](https://twitter.com/MoonbeamNetwork/status/1602625700569223169) to engage the community and attract users
+* Worked with DTMB & Moonwell to establish Moonbeam’s DeFi Voyage NFT campaign to spur activity within the ecosystem
+* Worked with parachains such as Acala, Phala, Astar to launch their native tokens on Moonbeam
+
+__CONNECTED CONTRACTS USE CASE__ (Updated Feedback from Turritz)
+This is something we’ve been EXTREMELY EXCITED about! For context, we’ve been working with Squid since last quarter of 2022 to lay the infrastructure + provide optimizations for crosschain swaps (CCS). Kudos to the Squid team for their thought-leadership and responsiveness in ensuring CCS was perfect. The coordinated launch of CCS swaps on Moonbeam was a huge landmark for the ecosystem, and StellaSwap was the first to launch CCS powered by Squid & Axelar. The importance of CCS in abstracting complexities and simplifying DeFi was the main reason why we were extremely enthusiastic about CCS and pushed forward a slew of targeted marketing campaign (e.g. with Moonbeam, Polygon, Arbitrum) to drive adoption and capital inflow towards Moonbeam. We’re proud to say that CCS on Moonbeam has been a great success, with Moonbeam leading the charts on usage and capital inflows:
+
+![image](upload://oOQ764w6rEzOrGDbYk147ohuYpD)
```

Comment:
Can you add the image to the PR - otherwise this will show a blank image placeholder.